StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Utils/LinkURLGenerator.swift

```diff
@@ -133,6 +133,22 @@ class LinkURLGenerator {
     }
 }
 
+// Used to get deterministic ordering for FundingSource tests
+extension Set where Element == LinkSettings.FundingSource {
+    func toSortedArray() -> [LinkSettings.FundingSource] {
+        return self.sorted { a, b in
+            switch (a, b) {
+            case (.card, .bankAccount):
+                return true
+            case (.bankAccount, .card):
+                return false
+            default:
+                return false
+            }
+        }
```

Comment:
Do we specifically want `card` before `bankAccount` here, or could we sort alphabetically by raw value? It would future-proof this logic for any new funding source that gets added (you never know! 😅)

```suggestion
        return self.sorted { a, b in
            a.rawValue.localizedCaseInsensitiveCompare(b.rawValue) == .orderedAscending
        }
```

Reply:
Sounds good, but will need to also update the test to reflect this.